WordPress/Classes/ViewRelated/Post/AbstractPostMenuHelper.swift

```diff
@@ -112,7 +112,7 @@ extension AbstractPostButton: AbstractPostMenuAction {
     func title(for post: AbstractPost) -> String {
         switch self {
         case .retry: return Strings.retry
-        case .view: return Strings.view
+        case .view: return post.status == .draft ? Strings.preview : Strings.view
```

Comment:
We want scheduled posts to display "Preview" as well
```suggestion
        case .view: return post.status == .published ? Strings.view : Strings.preview
```

<img width="1194" alt="Screenshot 2023-11-07 at 21 44 45" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/64531a78-dc4f-49f1-b4f7-73856e531dd3">

Reply:
Updated, thanks!